bin/configure.sh

```diff
@@ -27,3 +27,82 @@ export NODE_PATH=../..:../../zlux-server-framework/node_modules:$NODE_PATH
 __UNTAGGED_READ_MODE=V6 $NODE_BIN initInstance.js
 
 APP_WORKSPACE_DIR=${INSTANCE_DIR}/workspace/app-server
+
+# Add static definition for zss. TODO: Needs routedServices + apiInfo
+cat <<EOF >${STATIC_DEF_CONFIG_DIR}/zss.ebcidic.yml
+#
+services:
+  - serviceId: zss
+    title: Zowe System Services Server (ZSS)
+    description: Zowe System Services Server (ZSS) is used for enabling low-level microservices and other privileged services
+    catalogUiTileId: zss
+    instanceBaseUrls:
+      - https://${ZOWE_EXPLORER_HOST}:${ZOWE_ZSS_SERVER_PORT}/
+    homePageRelativeUrl:
+    routedServices:
+      - gatewayUrl: /server/agent/environment # Displays ZSS host environment info
+        serviceRelativeUrl: /server/agent/environment 
+      - gatewayUrl: /server/agent/services  # Returns list of ZSS services
+        serviceRelativeUrl: /server/agent/services
+      - gatewayUrl: /plugins
+        serviceRelativeUrl: /plugins
+      - gatewayUrl: /datasetMetadata/
+        serviceRelativeUrl: /datasetMetadata/
+      - gatewayUrl: /datasetContents/
+        serviceRelativeUrl: /datasetContents/
+      - gatewayUrl: /security-mgmt/classes/
+        serviceRelativeUrl: /security-mgmt/classes/
+      - gatewayUrl: /security-mgmt/access/
+        serviceRelativeUrl: /security-mgmt/access/
+      - gatewayUrl: /security-mgmt/groups/
+        serviceRelativeUrl: /security-mgmt/groups/
+      - gatewayUrl: /security-mgmt/user-profiles/
+        serviceRelativeUrl: /security-mgmt/user-profiles/
+      - gatewayUrl: /saf-auth/
+        serviceRelativeUrl: /saf-auth/
+      - gatewayUrl: /VSAMdatasetContents/
+        serviceRelativeUrl: /VSAMdatasetContents/
+      - gatewayUrl: /unixfile/contents
+        serviceRelativeUrl: /unixfile/contents
+      - gatewayUrl: /unixfile/chmod/
+        serviceRelativeUrl: /unixfile/chmod/
+      - gatewayUrl: /unixfile/chtag/
+        serviceRelativeUrl: /unixfile/chtag/
+      - gatewayUrl: /unixfile/chown/
+        serviceRelativeUrl: /unixfile/chown/
+      - gatewayUrl: /unixfile/metadata/
+        serviceRelativeUrl: /unixfile/metadata/
+      - gatewayUrl: /unixfile/touch/
+        serviceRelativeUrl: /unixfile/touch/
+      - gatewayUrl: /unixfile/mkdir/
+        serviceRelativeUrl: /unixfile/mkdir/
+      - gatewayUrl: /unixfile/copy/
+        serviceRelativeUrl: /unixfile/copy/
+      - gatewayUrl: /unixfile/rename/
+        serviceRelativeUrl: /unixfile/rename/
+      - gatewayUrl: /login/
+        serviceRelativeUrl: /login/
+      - gatewayUrl: /logout/
+        serviceRelativeUrl: /logout/
+      - gatewayUrl: /ras/
+        serviceRelativeUrl: /ras/
+      - gatewayUrl: /password/
+        serviceRelativeUrl: /password/
+      - gatewayUrl: /omvs/
+        serviceRelativeUrl: /omvs/
+      - gatewayUrl: /unixfile/
+        serviceRelativeUrl: /unixfile/
```

Comment:
why do we have a zillion routes?
this just wont work for static because there would be no way to register plugin routes which I don't even see on here.

You could just have 1 route
gatewayUrl: /
serviceRelativeUrl: /
why not?

Reply:
you may want to show the before & after as to what benefit there is to having so many routes, because if you did need all the routes + plugins then you'd need to add /ZLUX/plugins